File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/AttributeValueId.java
package org.dddml.wms.domain;


public class AttributeValueId
{
    private String attributeId;

    public String getAttributeId()
    {
        return this.attributeId;
    }

    public void setAttributeId(String attributeId)
    {
        this.attributeId = attributeId;
    }

    private String value;

    public String getValue()
    {
        return this.value;
    }

    public void setValue(String value)
    {
        this.value = value;
    }

    AttributeValueId ()
    {
    }

    public AttributeValueId (String attributeId, String value)
    {
        this.attributeId = attributeId;
        this.value = value;
    }

}

File: Dddml.Wms.AdminUI/ts/AttributeValueId.ts
namespace Dddml.Wms.Domain {

	export class AttributeValueId {

		private attributeId: string;

		getAttributeId(): string {
			return this.attributeId;
		}

		setAttributeId(attributeId: string) {
			this.attributeId = attributeId;
		}

		private value: string;

		getValue(): string {
			return this.value;
		}

		setValue(value: string) {
			this.value = value;
		}

	}
}

